db/db_impl_compaction_flush.cc

```diff
@@ -1022,7 +1023,7 @@ Status DBImpl::CompactFilesImpl(
 
   if (status.ok()) {
     // Done
-  } else if (status.IsShutdownInProgress()) {
+  } else if (status.IsShutdownInProgress() || status.IsColumnFamilyDropped()) {
```

Comment:
Can `status` be `IsShutdownInProgress`? Looks like it cannot be because it is the return value of `LogAndApply`.